### ISSUE TITLE
Where is the smooth part 

### ISSUE THREAD

**Issue by zlj0711** [yanx27/EverybodyDanceNow_reproduce_pytorch#70]
the loss function consider about the time smooth in the paper, but I cant find this part in the training code? Looking forward to some help.
